.github/workflows/release.yml

```diff
@@ -174,6 +174,16 @@ jobs:
       PYPI_API_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
       TEST_PYPI_API_TOKEN: ${{ secrets.TEST_PYPI_API_TOKEN }}
 
+  docker-release:
+    name: Docker Release
+    needs: [pypi-release]
+
+    uses: dbt-labs/dbt-release/.github/workflows/docker-release.yml@main
+    with:
+      package: ${{ github.event.repository.name }}
```

Comment:
This is not true for `dbt-postgres`. But the release workflow releases both at the same time. So I think this should be two jobs and the package name is hardcoded. That being said, that will fail for versions 1.8+.

Reply:
Good point - I rewrote this with a conditional based on minot version